src/creature.cpp

```diff
@@ -249,7 +249,7 @@ bool Creature::getNextStep(Direction& dir, uint32_t&)
 
 void Creature::startAutoWalk()
 {
-	Player* player = getPlayer();
+	auto player = getPlayer();
```

Comment:
Could this be `const auto`
```suggestion
	const auto player = getPlayer();
```